src/Zoomba/DesktopLibrary.py

```diff
@@ -519,34 +519,42 @@ def send_keys(self, *argv):
 
         A list of special key codes can be found
         [https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.keys.html|here]
+
```

Comment:
Just curious, since we are already using ActionChains, would calling key_up in lieu or reset_actions resolve having to send the modifier keys a second time to release?

Reply:
I gave it a try while working on it but:
1. It wasn't working for some reason, the key was always stuck. Given I never tried it with the reset actions so that may of made it work...
2. If i used key down and key up I would need to either make a separate keyword for using modifiers or programtically determine if we are a modifier

I think this just keeps it nice and simple for now. Im sure there is room for improvement still once people use it.